docs/notebooks/108-gpu-device-with-output.rst

```diff
@@ -246,7 +246,7 @@ for that property.
     GPU_QUEUE_THROTTLE            : Priority.MEDIUM
     GPU_ENABLE_LOOP_UNROLLING     : True
     CACHE_DIR                     : 
-    PERFORMANCE_HINT              : PerformanceMode.UNDEFINED
+    PERFORMANCE_HINT              : PerformanceMode.THROUGHPUT
```

Comment:
I suppose default is `LETANCY` for all our devices

Reply:
yes, plugin default mode is LATENCY.